go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/looplab/fsm
+module github.com/MonochromeHousing/fsm
 
 go 1.16
```

Comment:
I think it would be good to upgrade to latest version which is 1.22.5

Reply:
You mean go version (not fsm version I assume) ?

Reply:
Done